scripts/supervisord/run

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+set -eo pipefail
+
+# HACK. For some reason, *somewhere* $PATH is being overriden and the default
+# "/opt/rh/" is not being added. We instead manually search for the binaries and add it
+# to $PATH.
+for i in $(find /opt/rh/ -name 'bin' -type d | grep '/root/usr/bin'); do
```

Comment:
this is weird, we shouldn't need it.

Reply:
Yeah.. It's a hack right now since I was debugging the $PATH issues. (since we don't copy files over from `/opt/app-root` path.